src/org/plovr/ConfigOption.java

```diff
@@ -208,6 +208,13 @@ public boolean update(String prettyPrintParam, Config.Builder builder) {
     }
   }),
 
+  ENABLE_AGGRESSIVE_RAW_CACHING("enable-aggressive-raw-caching", new ConfigUpdater() {
```

Comment:
maybe add some comments on what this does, or something to https://github.com/bolinfest/plovr/blob/f7aceab0dce70e46c8c670b5d82d04c3ac092cbc/src/org/plovr/options.soy ?